database/models/events.db.php

```diff
@@ -117,13 +118,17 @@ class Event
     #[ManyToMany(targetEntity: UserGroup::class, inversedBy: 'events')]
     public Collection $groups;
 
+    #[OneToOne(targetEntity: Conversation::class, inversedBy: 'event', cascade: ["remove"])]
+    public Conversation $conversation;
```

Comment:
This might break existing events since they don't have a conversation.

Reply:
Added a null default, and when going to an old event if there is no conversation it creates one.